core/commands/add.go

```diff
@@ -81,6 +81,11 @@ You can now refer to the added file in a gateway, like so:
 			return nil
 		}
 
+		_, found, _ := req.Option(progressOptionName).Bool()
```

Comment:
That might be a good idea, except it changes the http api. which i'm explicitly trying to avoid right now.